src/Screens/HomeScreen/HomeScreen.tsx

```diff
@@ -8,26 +8,42 @@ import { useTranslation } from "react-i18next";
 import { fetchTodos } from "store/todo/todoActions";
 import { useDispatch } from "react-redux";
 import Form from "components/Form/Form";
+import { StyleSheet } from "aphrodite";
+import CustomButton from "components/CustomButton/CustomButton";
 
 const style = bemCssModule(HomeScreenStyles);
 
 const HomeScreen = () => {
   const { t } = useTranslation();
   const dispatch = useDispatch();
-  const btnAddTask = <button>Add task</button>;
+  const [showForm, setShowForm] = useState(false);
 
   useEffect(() => {
     dispatch(fetchTodos());
   }, [dispatch]);
 
   return (
     <LayoutWrapper>
-      <HeaderPanel actionButton={btnAddTask} />
+      <HeaderPanel />
+      <CustomButton
+        label="Add task"
+        onClick={() => setShowForm(!showForm)}
+        filled={false}
+        disabled={false}
+        customStyles={styles.btn}
+      />
       <h1 className={style("title")}>{t("active-tasks")}</h1>
-      <Form />
+      {showForm && <Form />}
       <Todo />
     </LayoutWrapper>
   );
 };
 
+const styles = StyleSheet.create({
+  btn: {
+    border: "1px solid blue",
+    color: "blue",
+  },
+});
+
```

Comment:
It is very good that customStyles prop is working, but remember that we want to keep all default styles in CustomButton component and only in some edge cases we want to overwrite them :)